package.json

```diff
@@ -102,24 +102,35 @@
     ],
     "rules": {
       "angular/angularelement": "error",
-      "angular/di": "error",
-      "angular/di-order": "error",
+      "angular/di": "warn",
+      "angular/di-order": "warn",
       "angular/di-unused": "error",
       "angular/document-service": "error",
       "angular/log": "error",
       "angular/module-dependency-order": "error",
       "angular/no-service-method": "error",
-      "angular/no-services": "error",
+      "angular/no-services": "warn",
       "angular/window-service": "error",
       "compat/compat": "error",
       "arrow-parens": "off",
       "constructor-super": "off",
       "generator-star-spacing": "off",
+      "max-len": [
+        "error",
+        {
+          "code": 120,
```

Comment:
This one's my own subjective opinion, I welcome a challenge to this.  I feel that a line length of 80 is far too short for programming today.  In recent years we've shifted to value descriptive function and variable names.  Combine that with the base level of indentation in files, and I believe this rule becomes unnecessarily restrictive to developers.  I feel that upping the line length by 40 to 120 gives developers room to breathe, but keeps files to a reasonable width.